go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ava-labs/teleporter-token-bridge
 
-go 1.21
+go 1.21.9
```

Comment:
Does setting the patch version in `go.mod` work now? I'm pretty sure it was breaking something before

Reply:
Yep it does! I learned this this morning when I saw https://github.com/ava-labs/avalanchego/pull/2938

Reply:
I believe this should also fix this warning:
![image](https://github.com/ava-labs/teleporter-token-bridge/assets/55204436/e90af46a-42d1-4b05-b412-a09d524bb895)

Reply:
Amazing!!